hivemind/utils/mpfuture.py

```diff
@@ -101,213 +134,133 @@ def _set_event_if_necessary(self):
         async def _event_setter():
             self._aio_event.set()
 
-        if self._loop.is_running() and loop == self.get_loop():
+        if self.get_loop().is_running() and loop == self.get_loop():
```

Comment:
nit: I'd suggest to:

1. Remove the `self.get_loop()` method and just use `self._loop`
2. Rename the local variable `loop` to `running_loop`

This would make clear that `self._loop` is this MPFuture's event loop and `running_loop` is the current running event loop.

Reply:
done that